flink-metrics/flink-metrics-influxdb/src/main/java/org/apache/flink/metrics/influxdb/InfluxdbReporterOptions.java

```diff
@@ -36,6 +36,12 @@
 		.noDefaultValue()
 		.withDescription("the InfluxDB server host");
 
+	public static final ConfigOption<String> SCHEME = ConfigOptions
+		.key("scheme")
+		.stringType()
```

Comment:
We could introduce a `Scheme` enum which defines the valid values `http`/`https`.